package.json

```diff
@@ -2,7 +2,7 @@
 	"name": "apache-daffodil-vscode",
 	"displayName": "Apache Daffodil VS Code Extension",
 	"description": "VS Code extension for Apache Daffodil DFDL schema debugging",
-	"version": "1.2.0-SNAPSHOT",
+	"version": "1.3.0-SNAPSHOT",
```

Comment:
Did we forget to remove the `-SNAPSHOT` from the 1.2.0 release? I thought the release candidate script checked for that and wouldn't allow a build?

Reply:
It was properly set to 1.2.0 for the release.  The issue was with merging this https://github.com/apache/daffodil-vscode/commit/3d98b6e17a20537325e5bb84b7c9d81225f80117.  Once this PR is merged we'll be good again.

Reply:
@Shanedell, can you give this PR a review?  Thanks!